mmv1/third_party/terraform/website/docs/ephemeral-resources/service_account_key.html.markdown

```diff
@@ -0,0 +1,45 @@
+---
+subcategory: "Cloud Platform"
+description: |-
+  Get a Google Cloud Platform service account Public Key
+---
+
+# google_service_account_key
+
+Get an ephemeral service account public key. For more information, see [the official documentation](https://cloud.google.com/iam/docs/creating-managing-service-account-keys) and [API](https://cloud.google.com/iam/reference/rest/v1/projects.serviceAccounts.keys/get).
+
+## Example Usage
+
+```hcl
+resource "google_service_account" "myaccount" {
```

Comment:
we could simplify the example since you technically don't need all these resources if you already know the id/name of the service account and the key that you desire to obtain.

We can also just leave as is to show how to set it up for ephemeral use in tfconfig.

Reply:
I think leaving it as it is would help users more - they have something more complete to understand / to copy-paste into their configs 😉